test/services/helpers/getAllRoutes.js

```diff
@@ -0,0 +1,40 @@
+const app = require('../../../src/app');
+
+const getAllRoutes = () => {
+	const convert = {};
+	let paths;
+	try {
+		({ docs: { paths } } = app);
+	} catch (err) {
+		throw new Error(err);
+	}
+
+	Object.keys(paths).forEach((path) => {
+		const methods = Object.keys(paths[path]);
+		const modle = path.indexOf('/{_id}') > -1;
+
+		if ( !(path.includes('/{_id}') || path.includes('/{id}')) ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before this paren.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5361392)